src/renderer/common/css/dialog.css

```diff
@@ -43,6 +43,11 @@ body {
     font-size: 1.5em;
 }
 
+.fl-dialog .fl-dialog-content .fl-dialog-contentText .fl-subhead {
+    margin: -12 0 10 0;
```

Comment:
I am not a great fan of using negative margins unless they are absolutely necessary. In this case you can simply reduce the bottom margin of the `fl-title`. Also, omitting the unit is acceptable only when the value is 0, i.e. add "px".